recipes/casadi/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "casadi" %}
+{% set version = "3.1.1" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/casadi/casadi.git
+  git_rev: {{ version }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - cmake
+    - python
+    - swig
+    - numpy >=1.10
```

Comment:
Sounds good. I had this in previous attempts but dropped it when I realized the official binaries don't specify a numpy version. (Also was getting some errors with numpy 1.9 in python 3.6, probably earlier version of the build script).

checking it locally before merging ... this is gonna take forever on CI :smile: